internal/clients/connect/client_connect.go

```diff
@@ -486,3 +486,46 @@ func (c *ConnectClient) ValidateDeployment(contentID types.ContentID, log loggin
 	}
 	return err
 }
+
+// Handle preflight agent error details
```

Comment:
the `preflightAgentError` code is a port from the previous implementation within the publish.go file.